### DISCUSSION TITLE
Having something like, Module system for solid-start

### DISCUSSION THREAD

**Discussion by q1b** [solidjs/solid-start#209] [General]
[link to nuxt modules page](https://modules.nuxtjs.org/)

anyway, any good reasons to not implement something like this ?

**Comment by nksaraf** [comment]
No good reason not to implement this, but we definitely want to design it a bit more and see what makes sense in a `vite` environment. Nuxt has to figure out how make things work across multiple builders, we just have one which is pretty extendible on its own. 

The main thing to figure out is what kind of lifecycle hooks does it make sense to provide.

**Reply by q1b** [reply]
Thanks for the reply, I understand not much but It seem you have covered it already in the list.
